sopel/test_tools.py

```diff
@@ -259,3 +272,257 @@ def run_example_tests(filename, tb='native', multithread=False, verbose=False):
         args.extend(["-n", str(cpu_count())])
 
     pytest.main(args)
+
+
+def botfactory(settings, autoloads=None):
```

Comment:
I have to say, it's weird that this is a function (lowercase name) and not a class (CamelCase name), just because the other stuff related to it is all classes.

Reply:
Yeah, it *feels* weird. I think if I add the pytest plugin within Sopel, I might figure a way to make names more consistent which will feel better to use.